app/Http/Requests/ValidateLength.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class ValidateLength extends FormRequest
```

Comment:
Please use `ValidateReminderLength` to avoid class name collision if one day we have other Length validators